WordPressVIPMinimum/Sniffs/Functions/DynamicCallsSniff.php

```diff
@@ -120,10 +120,6 @@ private function collect_variables() {
 			return;
 		}
 
-		if ( $this->tokens[ $t_item_key ]['length'] !== 1 ) {
```

Comment:
Just curious - what tokens are `==` and `===`?

Reply:
* `==` => `T_IS_EQUAL`
* `===` => `T_IS_IDENTICAL`

Ref: https://www.php.net/manual/en/tokens.php